wikimedia/wikipedia-ios
# Removed summary cache workaround

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T252845

Unfortunately I wasn't able to test this in the Beta Cluster, it continues to return the old content after a successful POST.

### Test Steps
1. Save article with a Wikidata title description
2. Go to article from Saved tab, note the saved tab cell subtitle.
3. Go to article, change the title description, wait for article to refresh.
4. Confirm you see your new title description in the article content
5. Go back to Saved, confirm you see the new title description in the cell subtitle.
